R/pkg/tests/fulltests/test_sparkSQL.R

```diff
@@ -3990,12 +3990,21 @@ test_that("Call DataFrameWriter.load() API in Java without path and check argume
   expect_error(read.df(source = "json"),
                paste("Error in load : analysis error - Unable to infer schema for JSON.",
                      "It must be specified manually"))
-  expect_error(read.df("arbitrary_path"), "Error in load : analysis error - Path does not exist")
-  expect_error(read.json("arbitrary_path"), "Error in json : analysis error - Path does not exist")
-  expect_error(read.text("arbitrary_path"), "Error in text : analysis error - Path does not exist")
-  expect_error(read.orc("arbitrary_path"), "Error in orc : analysis error - Path does not exist")
+  expect_error(read.df("arbitrary_path"),
+               paste("Error in load : analysis error - [PATH_NOT_FOUND] Path does not exist:",
+                     "file:/__w/spark/spark/arbitrary_path."), fixed = TRUE)
```

Comment:
@itholic can we avoid checking the exact path matching here? If we run the tests in other environmetns, they will always file.

Reply:
This actually breaks the CI in AppVeyor https://ci.appveyor.com/project/ApacheSoftwareFoundation/spark/builds/45304768. I am reverting this for now.